doc/creating_planner_plugin/emptyplan_planner_tutorial.rst

```diff
@@ -0,0 +1,42 @@
+Creating Planner Plugin Template
```

Comment:
At one point I suggested, and I believe @mlautman might like this better, that this tutorial be renamed to "Creating MoveIt Plugins" and it have sections for all [types of plugins](https://moveit.ros.org/documentation/plugins/). Many of them would be TODO for now, though.

Reply:
done